<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>port</class>
 <widget class="QDialog" name="port">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>port</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="12" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="rule_edit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;将生成命令复制到usb.rules&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>udev命令：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>生成udev命令</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>命令信息：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="KERENEL"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="see_kernel">
       <property name="text">
        <string>查看串口内核</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="usb_check">
        <property name="text">
         <string>USB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="acm_check">
        <property name="text">
         <string>ACM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>输入串口别名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="set_portname"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="udev_btn">
       <property name="text">
        <string>重启udev规则</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>查看更改情况</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="change_btn">
     <property name="text">
      <string>打开usb.rules文件并编辑</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>活跃串口列表：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serial_list"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>串口数量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="num_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="refresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLineEdit" name="password">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用与sudo命令&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>管理员密码：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="clear">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
